vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -285,26 +272,31 @@ type::ExternalWalletPtr GenerateLinks(type::ExternalWalletPtr wallet) {
 }
 
 std::string GenerateVerifyLink(type::ExternalWalletPtr wallet) {
-  std::string url;
+  std::string url{};
```

Comment:
Nope, not necessary. Most of the times I use braced initialization to default-construct objects. I got myself used to it, so that I can save myself a couple of uninitialized variables and undefined behaviors. 🙂